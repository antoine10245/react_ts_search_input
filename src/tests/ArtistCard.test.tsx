import * as React from "react";
import { shallow } from "enzyme";
import ArtistCard from "../components/ArtistCard.component";

const mockArtist = {
  id: 123,
  name: "hallo",
  url: "hallo",
  image_url: "hallo",
  thumb_url: "hallo",
  facebook_page_url: "hallo",
  mbid: "hallo",
  tracker_count: 123,
  upcoming_event_count: 123
};

describe("ArtistCard component", () => {
  it("renders without crashing", () => {
    const subject = shallow(<ArtistCard artist={mockArtist} />);
    expect(subject).toMatchSnapshot();
  });
});
