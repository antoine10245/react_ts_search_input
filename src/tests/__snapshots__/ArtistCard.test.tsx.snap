// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistCard component renders without crashing 1`] = `
<styled.div
  className="col-xs-12 col-lg-4 mb-4"
>
  <styled.div
    className="card"
  >
    <img
      alt="Card Aartist cap"
      className="card-img-top"
      src="hallo"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        hallo
      </h5>
      <p
        className="card-text"
      >
        hallo
      </p>
      <a
        className="btn btn-primary"
        href="hallo"
      >
        Go to facebook
      </a>
    </div>
  </styled.div>
</styled.div>
`;
